ask/models.py

```diff
@@ -31,6 +31,8 @@ class question(models.Model):
     points = models.IntegerField(default=1)
     solved = models.BooleanField(default=False)
     tags = models.ManyToManyField(tag, blank=True)
+    flair_icon = models.CharField(max_length=25, null=True)
```

Comment:
I don't get the need of the _flair_icon_ attribute in the model? 
It's  gonna be the same icon for everything right?